src/wp-includes/widgets/class-wp-widget-custom-html.php

```diff
@@ -202,44 +196,22 @@ public function update( $new_instance, $old_instance ) {
 	/**
 	 * Loads the required scripts and styles for the widget control.
 	 *
-	 * @since 4.9.0
+	 * @since CP-2.3.0
 	 */
 	public function enqueue_admin_scripts() {
-		$settings = wp_enqueue_code_editor(
-			array(
-				'type'       => 'text/html',
-				'codemirror' => array(
-					'indentUnit' => 2,
-					'tabSize'    => 2,
-				),
-			)
-		);
+		if ( ! isset( $this->number ) ) {
+			return;
+		}
 
+		wp_enqueue_code_editor( array( 'type' => 'text/html' ) );
+		wp_enqueue_script( 'wp-codemirror' );
 		wp_enqueue_script( 'custom-html-widgets' );
-		if ( empty( $settings ) ) {
-			$settings = array(
-				'disabled' => true,
-			);
-		}
-		wp_add_inline_script( 'custom-html-widgets', sprintf( 'wp.customHtmlWidgets.init( %s );', wp_json_encode( $settings ) ), 'after' );
-
-		$l10n = array(
-			'errorNotice' => array(
-				/* translators: %d: Error count. */
-				'singular' => _n( 'There is %d error which must be fixed before you can save.', 'There are %d errors which must be fixed before you can save.', 1 ),
-				/* translators: %d: Error count. */
-				'plural'   => _n( 'There is %d error which must be fixed before you can save.', 'There are %d errors which must be fixed before you can save.', 2 ),
-				// @todo This is lacking, as some languages have a dedicated dual form. For proper handling of plurals in JS, see #20491.
-			),
-		);
-		wp_add_inline_script( 'custom-html-widgets', sprintf( 'jQuery.extend( wp.customHtmlWidgets.l10n, %s );', wp_json_encode( $l10n ) ), 'after' );
 	}
 
 	/**
 	 * Outputs the Custom HTML widget settings form.
 	 *
-	 * @since 4.8.1
-	 * @since 4.9.0 The form contains only hidden sync inputs. For the control UI, see `WP_Widget_Custom_HTML::render_control_template_scripts()`.
+	 * @since CP-2.3.0
 	 *
 	 * @see WP_Widget_Custom_HTML::render_control_template_scripts()
```

Comment:
This line reference a function that has been removed.

Reply:
Go ahead and make that change along with the one below that you've suggested, @mattyrob .